.storybook/main.js

```diff
@@ -42,6 +43,20 @@ module.exports = {
       require.resolve('babel-plugin-remove-graphql-queries'),
     ];
 
+    config.module.rules.push({
+      test: /\.mdx?$/,
+      use: ['babel-loader', '@mdx-js/loader'],
+    });
+
+    // TODO: Figure out why Gatsby is throwing this error:
+    // 'The result of this StaticQuery could not be fetched & remove this alias.
+    ['.', '..', '../..', '../../..'].forEach((pathPrefix) => {
+      config.resolve.alias = {
+        ...config.resolve.alias,
+        [`${pathPrefix}/lib/useSiteMetadata`]: p.resolve(__dirname, './useSiteMetadata.js'),
+      };
+    });
```

Comment:
This is a hack for sure, but I don't really want to fiddle around with Gatsby anymore right now. It is slowing me down. This feels like an ok solution for the moment given it doesn't affect the primary app. If everyone is ok w/ it, I will file an issue for this to look into post-launch.